servatrice/src/servatrice.cpp

```diff
@@ -725,3 +725,11 @@ void Servatrice::doSendIslMessage(const IslMessage &msg, int serverId)
             interface->transmitMessage(msg);
     }
 }
+
+int Servatrice::getMaxUserLimit() const {
+    return settingsCache->value("security/max_users_total", 500).toInt();
+}
+
+bool Servatrice::getMaxUserLimitEnabled() const {
+    return settingsCache->value("security/enable_max_user_limit", false).toBool();
+}
```

Comment:
These helper functions would be better suited as members of the `SettingsCache` class